.gitignore

```diff
@@ -1,4 +1,10 @@
 coverage
-node_modules
+node_modules/*
 .DS_Store
 output
+
+#we must commit this single package to the repo as ncc cannot bundle it and the dist bundle is configure to use run-script as external package
+#yes, below looks strange but this is how it is done with git
+!node_modules/@npmcli/
+node_modules/@npmcli/*
+!node_modules/@npmcli/run-script/
```

Comment:
Is `!node_modules/@npmcli/run-script/` not enough?

Reply:
you didn't read my comment few lines above, did you? 😋

Reply:
I read and that I really wonder that need this construct 😄

Reply:
I've added [this](https://stackoverflow.com/questions/5533050/gitignore-exclude-folder-but-include-specific-subfolder) link to the comment